xml/System.Net/ServicePoint.xml

```diff
@@ -187,14 +187,17 @@
         <summary>Specifies the delegate to associate a local <see cref="T:System.Net.IPEndPoint" /> with a <see cref="T:System.Net.ServicePoint" />.</summary>
         <value>A delegate that forces a <see cref="T:System.Net.ServicePoint" /> to use a particular local Internet Protocol (IP) address and port number. The default value is <see langword="null" />.</value>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
 
 [!INCLUDE [obsolete-notice](~/includes/web-request-obsolete.md)]
 
- Some load balancing techniques require a client to use a specific local IP address and port number, rather than <xref:System.Net.IPAddress.Any?displayProperty=nameWithType> (or <xref:System.Net.IPAddress.IPv6Any?displayProperty=nameWithType> for Internet Protocol Version 6) and an ephemeral port. Your <xref:System.Net.ServicePoint.BindIPEndPointDelegate> can satisfy this requirement.  
-  
+ Some load balancing techniques require a client to use a specific local IP address and port number, rather than <xref:System.Net.IPAddress.Any?displayProperty=nameWithType> (or <xref:System.Net.IPAddress.IPv6Any?displayProperty=nameWithType> for Internet Protocol Version 6) and an ephemeral port. Your <xref:System.Net.ServicePoint.BindIPEndPointDelegate> can satisfy this requirement.
+
+> [!NOTE]
+>  On modern .NET, <xref:System.Net.HttpWebRequest> uses <xref:System.Net.Http.SocketsHttpHandler.ConnectCallback?displayProperty=nameWithType> to bind the underlying socket to the endpoint from this delegate.
```

Comment:
> modern .NET

I don't see us using this term anywhere else in the API docs. Moreover, these properties are unimplemented on .NET 8, which is still LTS. I would prefer us being specific here and elaborate the behavior per-version.

Reply:
Some of these were implemented in the past, some of them are there from the very beginning of .NET Core, some have been implemented this latest release. For that reason I chose the cop out of "Modern .NET" here. As figuring out what exact version to use for each of these is gonna be quite time consuming.
I can do that if you think it's worth the effort, but I don't really want to :smiling_face_with_tear:

Reply:
@gewarren any general guidelines about the terminlogy and the level of detail we should use here?

Reply:
I agree with Anton. "Modern .NET" doesn't provide clarity and will almost certainly lead to negative feedback for confused customers. Also, 10 years from now, the versions you're referring to won't be considered "modern .NET". Do you want me to try and help figure out the versions?

Reply:
As another cop out, I can just put .NET 9 everywhere. It's not wrong info, just in some cases it might be working for older versions as well.

> Do you want me to try and help figure out the versions?

Feel free to try, but just FYI, it's not about looking at the property itself, but at its usages in different classes (mainly `HttpWebRequest` AFAIK) and seeing if it's used in way that it will affect something.

Reply:
I'm fine putting .NET 9+ everywhere.